common/client/send_only_node_lifecycle.go

```diff
@@ -14,15 +13,17 @@ import (
 // It will continue checking until success and then exit permanently.
 func (s *sendOnlyNode[CHAIN_ID, RPC]) verifyLoop() {
 	defer s.wg.Done()
+	ctx, cancel := s.chStop.NewCtx()
```

Comment:
The utilities behind `chStop` seem useful outside of core. Are we thinking of moving them to `chainlink-common`?

Reply:
Yeah, already moved it a while back. Could phase out the core version now.

Reply:
Here is a PR to switch to aliases:
- https://github.com/smartcontractkit/chainlink/pull/11341